### PR TITLE
Added quoting for passwords in sample.app.ini and documentation

### PR DESCRIPTION
**Pull request by marove2000**
This pull request is addressing issue #3380.

I've added quote hints in sample.app.ini and it's documentation, preventing wrong behavior when parsing the file if special characters are used in passwords.

Without quotes the password will be read wrong.

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

File: docs/content/doc/advanced/config-cheat-sheet.en-us.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `HOST`: **127.0.0.1:3306**: Database host address and port.
 - `NAME`: **gitea**: Database name.
 - `USER`: **root**: Database username.
-- `PASSWD`: **\<empty\>**: Database user password.
+- `PASSWD`: **\`\<empty\>\`**: Database user password. Use \`your password\` for quoting, if you use special characters in the password.
```
Comment (strk) [inline docs/content/doc/advanced/config-cheat-sheet.en-us.md]: It doesn't look correct to quote `<empty>` ...

File: docs/content/doc/advanced/config-cheat-sheet.en-us.md [hunk 2]
```diff
@@ -185,7 +185,7 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `FROM`: **\<empty\>**: Mail from address, RFC 5322. This can be just an email address, or
    the "Name" \<email@example.com\> format.
 - `USER`: **\<empty\>**: Username of mailing user (usually the sender's e-mail address).
-- `PASSWD`: **\<empty\>**: Password of mailing user.
+- `PASSWD`: **\`\<empty\>\`**: Password of mailing user.  Use \`your password\` for quoting, if you use special characters in the password.
```
Comment (strk) [inline docs/content/doc/advanced/config-cheat-sheet.en-us.md]: quoting `<empty>` is unlikely correct, it's not a password, is an indication to the user that there's NOTHING written as a password...

**Review by strk — COMMENTED** [review]
Are you sure backtick quoting is generic ? Or is it only working for mysql backend ?

**Comment by lunny** [comment]
make L-G-T-M work

**Comment by marove2000** [comment]
I've removed the quotations from <empty> as suggested. Now it is only in the comment.

**Comment by thehowl** [comment]
Classic german comma before the dependent clause ;) But I'm not gonna nitpick to that, because that's an incredibly minor issue and the sentence still works. LGTM

**Pull request merged**